mixins/localize-mixin.js

```diff
@@ -128,6 +136,29 @@ export const LocalizeMixin = dedupeMixin(superclass => class extends superclass
 
 	}
 
+	localizeHTML(key, {
+		_link = '', _tooltip = '', _html = '',
+		_links = {}, _tooltips = {},
+		...replacements
+	} = {}) {
+		const localized = this.localize(key, replacements);
+		const sanitized = localized.replace(/</g, '&lt;').replace(/>/g, '&gt;');
+		const markedUp = sanitized
+			// replace [link], [link key], [/link]
+			.replace(/\[a(?: ?([\w\d-]*))?\]([^]*?)\[\/a\]/g, (m, n, t) => `<d2l-link ${_links[n] || _link}>${t}</d2l-link>`)
```

Comment:
I stuck with `a` for now. I don't have a strong opinion one way or the other, though.